python_modules/libraries/dagster-k8s/dagster_k8s/ops/k8s_job_op.py

```diff
@@ -228,20 +233,22 @@ def execute_k8s_job(
         namespace=namespace,
         resources=resources,
         scheduler_name=scheduler_name,
-        run_k8s_config={
-            "container_config": container_config,
-            "pod_template_spec_metadata": pod_template_spec_metadata,
-            "pod_spec_config": pod_spec_config,
-            "job_metadata": job_metadata,
-            "job_spec_config": job_spec_config,
-        },
+        run_k8s_config=UserDefinedDagsterK8sConfig.from_dict(
+            {
+                "container_config": container_config,
+                "pod_template_spec_metadata": pod_template_spec_metadata,
+                "pod_spec_config": pod_spec_config,
+                "job_metadata": job_metadata,
+                "job_spec_config": job_spec_config,
+            }
```

Comment:
i see `job_config` defined above but not here, is that right?

Reply:
Yeah job_config was intentionally excluded here and was probably a bad idea in the first place - every field on the job is already captured by some other field: https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/V1Job.md